atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -223,6 +164,51 @@ def _send_mail(self, receivers_list, subject, body):
             if s is not None:
                 s.quit()
 
+    def _get_receivers_list(self):
+        receivers_list = []
+        if self.submitter and self.submitter != self.DEFAULT_SUBMITTER:
+            receivers_list.append(self.submitter)
+
+        if self.to_koji_submitter or self.to_koji_pkgowner:
+            try:
+                metadata = get_build_json().get("metadata", {})
+                session = create_koji_session(self.koji_hub, self.koji_auth_info)
+            except Exception as e:
+                self.log.error("Failed to connect to koji:\n%s", repr(e))
+            else:
+                if self.to_koji_submitter:
+                    try:
+                        koji_task_id = metadata.get('labels', {}).get('koji-task-id')
+                        koji_task_info = session.getTaskInfo(koji_task_id)
+                        koji_task_owner = session.getUser(koji_task_info['owner'])
+                        koji_task_owner_email = koji_task_owner['krb_principal'].lower()
```

Comment:
Think it's probably better to take a param for `email_domain` which can be used to make up the user's email address.
Kerberos shouldn't be required for this.